test/gpx.js

```diff
@@ -2,6 +2,10 @@ import assert from 'assert';
 import fs from 'fs';
 import GPX from '../src/gpx';
 
+if (typeof require !== "undefined") {
+  var DOMParser = require("xmldom").DOMParser;
```

Comment:
It is not necessary to .be dependent on "xmldom".  Please remove the dependency and revise the new test case (you can refer to "Parse a gpx string" test case).

Reply:
I removed the dependency and changed the test accordingly